query/src/sql/exec/mod.rs

```diff
@@ -320,6 +337,7 @@ impl PipelineBuilder {
                 )
             })?;
 
+        self.pipeline.resize(1)?;
```

Comment:
Very critical, but previously missed, otherwise the final result is not aggregated

Reply:
I've added this for you in a previous PR, but it seems missed in some rebase operation. 😢